cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -40,46 +40,51 @@ In `{odo-title}`, services are provisioned from the OpenShift Service Catalog an
 [id="odo-supported-languages-and-images"]
 === Officially supported languages and corresponding container images
 
-.Supported languages, container images, and package managers
+.Supported languages, container images, package managers, and platforms
 [options="header"]
 |===
-|Language |Container image |Package manager
+|Language |Container image |Package manager |Platform
 |*Node.js*
-|https://registry.centos.org/centos/nodejs-10-centos7/latest[centos/nodejs-10-centos7]
-|NPM
-
-|
-|https://hub.docker.com/r/centos/nodejs-12-centos7[centos/nodejs-12-centos7]
+|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-10-rhel7[rhscl/nodejs-10-rhel7]
 |NPM
+|amd64, s390x
 
 |
 |https://catalog.redhat.com/software/containers/rhscl/nodejs-12-rhel7/[rhscl/nodejs-12-rhel7]
```

Comment:
I know you didn't change this link, but it doesn't work, if that's something you could look into

Reply:
I'll ask someone in the odo-team about the links.